<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{color: blue; font: bold 10pt;}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="placeholderText">
       <string>Введите текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; color:#ff0000;&quot;&gt;-&amp;gt; ТЕКСТ &amp;lt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Загрузить картинку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveImgButton">
      <property name="text">
       <string>Сохранить картинку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="encodeMsgButton">
      <property name="text">
       <string>Закодировать текст</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decodeMsgButton">
      <property name="text">
       <string>Декодировать текст</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
